examples/example_llama3.py

```diff
@@ -252,6 +253,14 @@ def _pass(graph):
     sharding_placement = autop.optimize_placement(verbose=True)
     print(f"Took {time.time() - t:.2f} s")
     parallel_mod = autop.apply_placement(sharding_placement)
+    multiplex_graph = True
+    if multiplex_graph:
+        f_gm = autop.fw_module
+        b_gm = autop.bw_module
+        multiplexed_gm = multiplex_fw_bw_graph(f_gm, b_gm)
+        print(f_gm.graph)
+        print(b_gm.graph)
+        print(multiplexed_gm.graph)
```

Comment:
Can we maybe start creating self-contained tests instead of overloading the `example_llama3.py`? I think it would enable to scale things a bit further.